shell/browser/api/atom_api_native_theme.cc

```diff
@@ -36,6 +37,10 @@ void NativeTheme::SetThemeSource(ui::NativeTheme::ThemeSource override) {
   // Update the macOS appearance setting for this new override value
   UpdateMacOSAppearanceForOverrideValue(override);
 #endif
+  const bool dark_enabled = ShouldUseDarkColors();
+  for (auto* window : electron::api::TopLevelWindow::GetAllWindows()) {
```

Comment:
You can iterate all windows with the `electron::WindowList` class. Also the `electron::` namespace prefix can be omitted here.